cmake/android_dependencies.cmake

```diff
@@ -65,6 +65,7 @@ set(FIREBASE_MESSAGING_ANDROID_DEPS
     "com.google.firebase:firebase-messaging:23.1.1"
     "com.google.firebase:firebase-analytics:21.2.0"
     "com.google.firebase:firebase-iid:21.1.0"
+    "com.google.flatbuffers:flatbuffers-java:1.12.0"
```

Comment:
Are we going to update the version if flatbuffer has a new release?

Reply:
We'll keep this in sync with whatever version the C++ repo is using.